src/collections/blog/2021-04-05-meshmate-of-the-year-2020-nikhil-ladha/meshmate-of-the-year.css

```diff
@@ -0,0 +1,3 @@
+.responsive__blog {
```

Comment:
This file is not being used right?

Reply:
It is being used in the meshmate-of-the-year mdx file

Reply:
But I don't see this file being imported anywhere 🤔

Reply:
Oh, the css file, it's no longer being used, I already put the styling in the general styles folder.

Also, noted the image review